src/main/java/org/junit/runner/JUnitCommandLineParseResult.java

```diff
@@ -15,11 +15,6 @@
     private final List<Throwable> parserErrors = new ArrayList<Throwable>();
 
     /**
-     * Do not use. Testing purposes only.
-     */
-    JUnitCommandLineParseResult() {}

```

Comment:
This is not redundant. It is a package-scope constror, preventing code outside of this package from constructing an instance via the constructor.